### PR TITLE
[stable/memcached] Add ability to specify arbitrary custom labels in memcached helm chart

### PR DESCRIPTION
**Pull request by orkhanM**
Signed-off-by: orkhanm <orkhan.mamedov.7@gmail.com>

#### What this PR does / why we need it:
This allows us to specify custom labels for the memcached statefulset, this is useful for defining custom alert and monitoring policies in Newrelic, Datadog, Prometheus, etc.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/memcached/templates/statefulset.yaml [hunk 1]
```diff
@@ -17,6 +17,9 @@ spec:
         chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
         release: "{{ .Release.Name }}"
         heritage: "{{ .Release.Service }}"
+        {{- range $key, $val := .Values.customLabels }}
```
Comment (olemarkus) [inline stable/memcached/templates/statefulset.yaml]: Could you use `with` like podAnnotations below?

File: stable/memcached/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: memcached
-version: 2.6.0
+version: 2.6.1
```
Comment (olemarkus) [inline stable/memcached/Chart.yaml]: This needs a minor version bump

**Review by olemarkus — CHANGES_REQUESTED** [review]

File: stable/memcached/Chart.yaml (hunk 2 above)
Comment (orkhanM) [reply stable/memcached/Chart.yaml]: :+1:

File: stable/memcached/templates/statefulset.yaml (hunk 1 above)
Comment (orkhanM) [reply stable/memcached/templates/statefulset.yaml]: @olemarkus I've added the requested changes, thanks for such a quick response!

File: stable/memcached/README.md [hunk 3]
```diff
@@ -54,6 +54,7 @@ The following table lists the configurable parameters of the Memcached chart and
 | `extraVolumes`            | Volume definitions to add as string | Un-set                                              |
 | `kind`                    | Install as StatefulSet or Deployment | StatefulSet                                        |
 | `podAnnotations`          | Map of annotations to add to the pod(s) | `{}`                                            |
+| `customLabels`            | Custom Labels to be applied to statefulset | Un-set                                   |
```
Comment (olemarkus) [inline stable/memcached/README.md]: I think podLabels is a better name here. This was also the key used by another chart for the same thing. Could you rename?

**Review by olemarkus — CHANGES_REQUESTED** [review]

File: stable/memcached/README.md (hunk 3 above)
Comment (orkhanM) [reply stable/memcached/README.md]: Sure thing

**Pull request merged**